com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStreamPlayer.cs

```diff
@@ -257,7 +262,11 @@ void OnEnable()
         void OnDisable()
         {
             if (abcStream != null)
+            {
                 abcStream.Dispose();
+            }
+
+            abcStream = null;
```

Comment:
It would make more sense to move it inside the brackets and set it only when its not already null.